csi/server/plugin/opensds/node.go

```diff
@@ -17,258 +17,424 @@ package opensds
 import (
 	"fmt"
 
-	"google.golang.org/grpc/codes"
-
+	"os/exec"
 	"strings"
 
 	csi "github.com/container-storage-interface/spec/lib/go/csi/v0"
 	"github.com/golang/glog"
 	"github.com/opensds/nbp/client/iscsi"
 	sdscontroller "github.com/opensds/nbp/client/opensds"
 	"github.com/opensds/nbp/driver"
+	c "github.com/opensds/opensds/client"
 	"github.com/opensds/opensds/pkg/model"
 	"golang.org/x/net/context"
+	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
 ////////////////////////////////////////////////////////////////////////////////
 //                            Node Service                                    //
 ////////////////////////////////////////////////////////////////////////////////
 
-func (p *Plugin) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRequest) (*csi.NodeStageVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+var (
+	// Client opensds client
+	Client *c.Client
```

Comment:
I downloaded this PR but can't even build it.

root@ubuntu:~/gopath/src/github.com/opensds/nbp# make
mkdir -p  /root/gopath/src/github.com/opensds/nbp/build/out
go build -o /root/gopath/src/github.com/opensds/nbp/build/out/csi.server.opensds github.com/opensds/nbp/csi/server
# github.com/opensds/nbp/csi/server/plugin/opensds
csi/server/plugin/opensds/node.go:41:2: Client redeclared in this block
	previous declaration at csi/server/plugin/opensds/controller.go:42:2